.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -6,7 +6,6 @@ on:
 
 jobs:
   update-and-publish:
-    if: ( !startsWith(github.event.head_commit.message, 'Auto update cromwell-helm chart version') )
```

Comment:
ah, I see.

I'm told that the "Ensure that a Jira ID is present" step is crucial from a compliance point of view... this may be a show-stopper for the quick fix